samples/tfm_integration/tfm_ipc/CMakeLists.txt

```diff
@@ -7,13 +7,23 @@ set(QEMU_KERNEL_OPTION "-device;loader,file=${CMAKE_BINARY_DIR}/tfm_qemu.hex")
 
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 
+
 # Add "tfm" as an external project via the TF-M module's cmake file
-trusted_firmware_build(BINARY_DIR ${CMAKE_BINARY_DIR}/tfm
-                       BOARD ${TFM_TARGET_PLATFORM}
-                       IPC
-                       CFGFILE "ConfigRegressionIPC"
-                       OUT_VENEERS_FILE VENEERS_FILE
-)
+if(${TFM_TARGET_PLATFORM} STREQUAL "STM_NUCLEO_L552ZE_Q")
+   trusted_firmware_build(BINARY_DIR ${CMAKE_BINARY_DIR}/tfm
+                          BOARD ${TFM_TARGET_PLATFORM}
+                          IPC
+                          CFGFILE "ConfigRegressionIPCTfmLevel2"
```

Comment:
What is the reason for using this configuration, with L2 isolation? I.e. why is it different from the default?

Reply:
@ioannisg as discussed, aim was to focus on Level2

Reply:
OK